### PR TITLE
Add survey link to add-on submission email

### PR DESCRIPTION
**Pull request by wagnerand**
We're currently running a developer survey, let's include it in the welcome email, to get more responses!

### CODE REVIEW

File: src/olympia/devhub/templates/devhub/email/submission.txt [hunk 1]
```diff
@@ -22,6 +31,6 @@ Add-ons help personalize the web experience for millions of users who have insta
 Happy developing!
 
 
--- 
+--
```
Comment (wagnerand) [inline src/olympia/devhub/templates/devhub/email/submission.txt]: SublimeText found and successfully exterminated a trailing space! 💪

Comment (diox) [reply src/olympia/devhub/templates/devhub/email/submission.txt]: The standard for email signatures is 2 dashes and a trailing space though :)

Comment (EnTeQuAk) [reply src/olympia/devhub/templates/devhub/email/submission.txt]: https://www.ietf.org/rfc/rfc3676.txt section 4.1

Comment (wagnerand) [reply src/olympia/devhub/templates/devhub/email/submission.txt]: 🧐 But...the text version does not have it either...

Comment (wagnerand) [reply src/olympia/devhub/templates/devhub/email/submission.txt]: Changed.

**Pull request merged**